Casks/q/quickapp-ide.rb

```diff
@@ -0,0 +1,26 @@
+cask "quickapp-ide" do
+  version "6.6.1"
+  sha256 "260be97a98b1b11aef430e82f0c35334b394953810a86f0988152fe21564dc17"
+
+  url "https://statres.quickapp.cn/quickapp/show/ide/quickapp-ide-#{version}.pkg"
+  name "quickapp-ide"
+  desc "Quickapp Development Tool"
+  homepage "https://www.quickapp.cn/"
+
+  livecheck do
+    url "https://www.quickapp.cn/api/ide/rpk/getPackage"
+    strategy :page_match
+    regex(/quickapp-ide-(\d+(?:\.\d+)*)\.pkg/i)
```

Comment:
```suggestion
    regex(/quickapp[._-]ide[._-]v?(\d+(?:\.\d+)+)\.pkg/i)
```
Minor regex tweaks to align with prevailing standards.